xcode/Subconscious/Shared/Services/Noosphere/Sphere.swift

```diff
@@ -178,6 +180,20 @@ public actor Sphere: SphereProtocol, SpherePublisherProtocol {
         logger.debug("init with identity: \(identity)")
     }
     
+    private static nonisolated func fetchIdentityFromSphere(noosphere: OpaquePointer, sphere: OpaquePointer) throws -> String {
```

Comment:
Initializing with a blocking call (`nonisolated` makes the function synchronous and blocking in the calling context) worries me after fixing other similar synchronous calls in #501. These seemingly innocuous calls caused UI freezes due to unexpected latencies caused by sync blocking.

Async initializers are not allowed in swift AFAIK. However, what if we instead created an async static factory function?

```swift
actor Sphere {
    // ...
    static func from(noosphere: Noosphere, sphere: OpaquePointer) async throws -> Self {
        // ...etc
        return self.init(noosphere: Noosphere, identity: identity)
    }
}
```

and at call site:

```swift
let sphere = await Sphere.from(noosphere: noosphere, sphere: pointer)
```

This prevents main thread from calling the code in a synchronous manner.